### DISCUSSION TITLE
실시간으로 변동하는 가격을 추적하기 위한 최적의 크롤링 전략 수립 방안

### DISCUSSION THREAD

**Discussion by lky3004me** [100-hours-a-week/discussions#6] [개발] [answered]
###  **문제점 & 현 상황**
현재 다나와에 대한 가격 변동을 추적하려고 합니다.
다만 밴을 당하지 않기 위해서, 다나와에서 크롤링을 수행하는 것이 아닌, 각 상품의 판매처 url로 크롤링을 수행하려고 합니다.

### **해결책**
 각 상품의 판매처 url로 크롤링을 수행하려고 합니다.
이 방법의 문제점은 각 상품의 판매처 별로 셀렉터가 다르기 때문에 적어도 24개의 사이트에서 크롤링을 수행해야 합니다.

24개 사이트
11번가, CJ온스타일, GS샵, G마켓, KT알파 쇼핑, NS몰, SK스토아, SSG.COM, YES24, 동원몰, 롯데ON, 롯데마트, 롯데홈쇼핑, 멸치쇼핑, 배민상회, 신세계라이브쇼핑, 신세계몰, 오늘의 집, 옥션, 이마트인터넷쇼핑몰, 쿠팡, 현대Hmall, 홈&쇼핑, 홈플러스

---
### **질문**
이 방법이 현실적인지 다른 방법을 찾아야할지 궁금합니다.

**Comment by ej31** [comment, marked as answer]
🤡
스크래핑 해야하는 데이터가 다나와에 포함되어 있기 때문에 말씀하신 24개 사이에 대한 스크래핑을 모두 진행하지 않아도 됩니다.
다만 리디렉션 되는 페이지의 끝점에 해당하는 링크는 확보해 두셔야 합니다.
